### ISSUE TITLE
Support for nested lists? 

### ISSUE THREAD

**Issue by jacoblippincott** [danomatic/react-pdf-html#30]
Does this package support nested lists? I have a <ul> nested inside a <ol> and in the rendered PDF I see that the bullet points are positioned in front of the unordered list items as expected but they are all on a single line. I can provide code and output if you would like.

**Comment by danomatic** [comment]
I don't think nested lists have been tested yet, but that shouldn't be very hard to solve. PRs welcome :-)

There may be a way to do a temporary fix for this with a stylesheet.

**Comment by TimHav** [comment]
As a temporary fix you can add some `padding-left` to the nested list. For example:

```html
<ul>
  <li>...</li>
  <ul style="padding-left: 30pt">
    <li>...</li>
    <li>...</li>
  </ul>
</ul>
```

**Comment by zbiggs1** [comment]
I'm also having the same issue. Puts the nested list's items inline. I can't seem to get CSS to jerryrig it either.

**Comment by danomatic** [comment]
@jacoblippincott this should be fixed in the latest version `v1.1.13`. It was caused by rendering list item content as inline/text
